### PR TITLE
new:usr:Improve RPM package with configuration and scripts

### PR DESCRIPTION
**Pull request by vrajat**
This commit adds 
* configuration files for bookkeeper and lds. 
* Start/Stop scripts
* Sets up run, log and conf directories.

Tested using rubix-admin on Qubole clusters.

### CODE REVIEW

File: rubix-rpm/src/main/resources/rpm/init.d/rubix.service [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  echo "$0 [start|stop]"
+  exit $2
+}
+
+start() {
+  export HADOOP_OPTS="-Dlog4j.configuration=file://${LOG4J_FILE}"
+  # Set the fs impls as CachingNativeS3FileSystem for book-keeper. HADOOP_CUSTOM_OVERRIDES are the best bet to do this without affecting all hadoop jobs
+  # HADDOP_CUSTOM_OVERRIDES are Qubole specific and the values is '!' separated key=value pair with value base64 encoded
+  # We also disable filesystem cache
+  export HADOOP_CUSTOM_OVERRIDES='fs.s3.impl.disable.cache=dHJ1ZQ==!fs.s3n.impl.disable.cache=dHJ1ZQ==!'
```
Comment (stagraqubole) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: this, HADOOP_CUSTOM_OVERRIDES,  needs to be removed

Comment (stagraqubole) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: We need to add setting config here to disable s3/s3n fs cache (fs.s3.impl.disable.cache) or recommend users to do it themselves in hadoop config as we have seen issues with cache enabled.

Comment (vrajat) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: Can you give me the command to do that ? Or shall I skip it and create an issue that this has to be documented ? 
I prefer automating vs documenting

Comment (stagraqubole) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: We have to find a way to get it done in OSS. We could find that out quickly hence resorted to qubole specific HADOOP_CUSTOM_OVERRIDES

Comment (vrajat) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: OK. Then I'll remove it and open an issue. Is that OK ?

Comment (stagraqubole) [inline rubix-rpm/src/main/resources/rpm/init.d/rubix.service]: Yes. Rest looks good to me

**Comment by vrajat** [comment]
Done. I've removed HADOOP_CUSTOM_OVERRIDES and created #62 @stagraqubole

**Pull request merged**